### PR TITLE
Update issue template for Product Owners at PUL

### PR DESCRIPTION
**Pull request by kelea99**
Just need a review for proposed changes

### CODE REVIEW

**Review by eliotjordan — APPROVED** [review]
Looks good. I'll wait for others to chime in before merging.

File: .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+_Please include hard deadlines, if the exhibit is part of an event, the issue is stopping work, etc._
+
+### Priority recommendation (Product Owner)
+
+- [ ] asap
+- [ ] within the next 3 weeks
+- [ ] PO will prioritize
+
+## Sudden Priority Justification (Product Owner)
+_Required if "asap" or "within the next 3 weeks" is checked. Add "Sudden Priority" and "Maintenance/Research" labels_
```
Comment (hackartisan) [inline .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md:25]: We have moved away from the maintenance/research board.

```suggestion
_Required if "asap" or "within the next 3 weeks" is checked. Add "Sudden Priority" and "dls-work-cycle" labels_
```

File: .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+### Implementation Notes
+_To be filled out by DLS._
+
+### Acceptance Criteria
+_To be filled out by DLS._
+
```
Comment (hackartisan) [inline .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md:32]: would it be confusing to add our "requirements to consider" here? see https://raw.githubusercontent.com/pulibrary/dpul-collections/refs/heads/main/.github/ISSUE_TEMPLATE/issue_template.md as an example

**Review by hackartisan — CHANGES_REQUESTED** [review]
Thank you for this! I left a couple of comments.

File: .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md (hunk 1 above)
Comment (kelea99) [reply .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md:25]: sounds good!

File: .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md (hunk 2 above)
Comment (kelea99) [reply .github/ISSUE_TEMPLATE/standard-issue-template-for-pos.md:32]: No reason not to try it! we will alter accordingly after feedback from folks!

**Review by hackartisan — APPROVED** [review]

**Pull request merged**